samples/managed-app-sample/createUiDefinition.json

```diff
@@ -0,0 +1,189 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+	"handler": "Microsoft.Azure.CreateUIDef",
+	"version": "0.1.2-preview",
+	"parameters": {
+		"config": {
+			"isWizard": false,
+			"basics": {
+				"location": {
+					"visible": "[basics('createNewCluster')]",
+					"allowedValues": [
+						"eastus",
```

Comment:
Currently we support a total of 33 regions. Can you please add all the 33 locations in all the CreateUIDef files?